.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: isort
         name: isort (python)
   - repo: https://github.com/psf/black
-    rev: 22.1.0
+    rev: 22.3.0
```

Comment:
The old version of black is not compatible with newer versions of click and due to that pre-commit fails to run - see more [here](https://github.com/psf/black/issues/2964).

Bumping version of black resolves this problem for me (and hopefully for others too).

I did try to run `pre-commit run --all-files` to make sure there are no differences in formatting and it works the same way as the old version (no files changed).